garage/envs/mujoco/mujoco_env.py

```diff
@@ -102,7 +101,7 @@ def observation_space(self):
 
     @property
     def action_bounds(self):
-        return bounds(self.action_space)
+        return self.action_space.low, self.action_space.high
```

Comment:
Some spaces does not have a high and a low. e.g. gym.Discrete

Reply:
Yeah not sure what to do about this. MuJoCo is continuous control, so discrete makes little sense. But conceivably someone could want to input a tuple space.

I think it's okay to leave the Box assumption as long as we assert it somewhere and nothing is using non-boxes yet. If someone hits the assert they can extend to support non-Box spaces.

Reply:
Or since we're not removing the `util.py` file, I can continue using `bounds`.

Reply:
I'll leave the code as-is for now and add some assert statements instead.

Reply:
Why do that if the definition of bounds() is just this code + NotImplementedError for non-Box spaces? You can already check that the action space is a Box.